### ISSUE TITLE
ngx-onsenui: Unexpected token: name (ComponentLoader) 

### ISSUE THREAD

**Issue by puku0x** [OnsenUI/OnsenUI#2063]
__Environment__ 

```
[Core]
    onsenui 2.3.3

[Framework]
    angular 4.1.3

[Framework binding]
    ngx-onsenui 4.0.0-rc.0

[Additional libraries]
    none

[Platform]
    Desktop - macOS Sierra

[Browser]
    Desktop - Chrome 55.0.2883.75
```

__Encountered problem__
Following error messages are showed when buliding an app with production mode.
```
ERROR in vendor.4c882dadfd376f5c7014.bundle.js from UglifyJs
Unexpected token: name (ComponentLoader) [vendor.4c882dadfd376f5c7014.bundle.js:9,6]
```

Is this related to https://github.com/OnsenUI/OnsenUI/commit/464cb6d81e2793319e03d8fa911631fbd3bc769f ?

__How to reproduce__
1. import `ngx-onsenui`
2. Build it with UglifyJS, for example `ng build --prod` or `webpack -p`

Reference:
https://github.com/puku0x/angular-cli-onsenui

**Comment by asial-matagawa** [comment]
@puku0x 
Thank you for reporting!
I fixed it and just released `ngx-onsenui@4.0.0-rc.1`.
https://github.com/OnsenUI/OnsenUI/blob/master/bindings/angular2/CHANGELOG.md#400-rc1

Closing this.

**Comment by puku0x** [comment]
@asial-matagawa Thank you for your quick response !
